dataactvalidator/filestreaming/loaderUtils.py

```diff
@@ -25,85 +28,62 @@ def compareRecords (recordA,recordB, fields) :
         return True
 
     @classmethod
-    def loadCsv(cls,filename,model,interface,fieldMap,fieldOptions):
-        """ Loads a table based on a csv
+    def cleanData(cls, data, model, fieldMap, fieldOptions):
+        """ Cleans up a dataframe that contains domain values
 
         Args:
-            filename: CSV to load
-            model: ORM object for table to be loaded
-            interface: interface to DB table is in
-            fieldMap: dict that maps columns of the csv to attributes of the ORM object
+            data = dataframe of domain values
+            fieldMap: dict that maps columns of the dataframe csv to our db columns
```

Comment:
I recognize that the `fieldMap` wasn't introduced in this PR, but it feels hairy. Is there a way to define those values statically instead of at method invocation time?